src/utils/index.js

```diff
@@ -1,4 +1,44 @@
-export { getViews } from "./getViews";
-export { getSortedVideos } from "./getSortedVideos";
-export { getFilteredCategory } from "./getFilteredCategory";
-export { getFilteredLanguage } from "./getFilteredLanguage";
+export const getSortedVideos = (videos, filters) => {
+  switch (filters.sort) {
+    case "MOST_RECENT":
```

Comment:
Keep type as constant's and import for use it would be clean and readable

Reply:
What I can see here is that type is `const` only.
and making such small(3-4 lines) files wasn't of much sense to me so I defined all utils in one file only which altogether is just 50 lines and I think it's still readable.